src/main/python/poetry.ts

```diff
@@ -156,6 +157,24 @@ export async function poetryInstallDepUserGroup(
   return true;
 }
 
+export async function poetryInstallRequirementsUserGroup(
+  filePath: string,
+): Promise<boolean> {
+  try {
+    const content = fs.readFileSync(filePath, { encoding: "utf-8" }).toString();
+    const packages: string[] = content
+      .split("\n")
+      .filter((line) => line !== "" && !line.startsWith("#"));
```

Comment:
I prefer all at once, because poetry checks the compatibility of the libraries being installed before installing them. If you go one by one, this check isn't done, and maybe 20 libraries have been installed and need to be changed because the 21th is not compatible

Reply:
+ this infers that the file is the equivalent of a `pip freeze > requirements.txt` file.